### PR TITLE
Added information for section 3.1 and 3.4 in the UG.

### PR DESCRIPTION
**Pull request by DonnyS57**
Added 3.1 and 3.4 sections for UG

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -86,15 +86,87 @@ This marks the end of the quick start. Click
 
 ## 3. About
 
+This section aims to help familiarise you with how our application works as well as the terminologies that we use
+throughout our application and user guide.
+
 ### 3.1 Special Icons
 
+Here are the special icons used throughout the User Guide to help you utilise the User Guide more effectively.
```
Comment (sheehui) [inline docs/UserGuide.md:94]: Here are the special icons used throughout our User Guide to help you utilise this guide more effectively.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -86,15 +86,87 @@ This marks the end of the quick start. Click
 
 ## 3. About
 
+This section aims to help familiarise you with how our application works as well as the terminologies that we use
+throughout our application and user guide.
```
Comment (sheehui) [inline docs/UserGuide.md:90]: hmm not sure if we need our application

File: docs/UserGuide.md [hunk 3]
```diff
@@ -86,15 +86,87 @@ This marks the end of the quick start. Click
...
+
+Tips are tidbits of helpful information that can help you have a better experience with DonnaFin.
+
+<div markdown="span" class="alert alert-primary">:bulb:
+Tip: Tips are helpful!
+</div><br>
+
+#### Notes
+
+Notes are here to provide the User with extra information that they may find useful when using DonnaFin.
```
Comment (sheehui) [inline docs/UserGuide.md:106]: provide you with extra information

File: docs/UserGuide.md [hunk 4]
```diff
@@ -86,15 +86,87 @@ This marks the end of the quick start. Click
...
+
+<div markdown="span" class="alert alert-primary">:information_source:
+Take notes when you see this icon!
+
+
+#### Caution 
+
+Cautions are around to warn users of potential unwanted side effects that may occur on certain parts of DonnaFin if the
```
Comment (sheehui) [inline docs/UserGuide.md:114]: warn you of potential unwanted side effects that may occur when using certain features of DonnaFin if you fail to follow this guide properly or tamper with the files of the application without adequate knowledge.

File: docs/UserGuide.md [hunk 5]
```diff
@@ -86,15 +86,87 @@ This marks the end of the quick start. Click
...
 
-### 3.4 Application Client Window Breakdown
+#### 3.3.4 Asset/Liabilities Tab
 
-### 3.5 Command Format
+###### This section covers both the asset and liability tab due to their similarities.
+
+The Asset/Liabilities tab is the Client View tab that stores all of your client's assets/liabilities. This tab can be
+accessed by pressing on the Assets/Liabilities tab when in any of the Client View tabs, or through typing the command
+`tab a`, `tab asset`, or `tab assets` for the Asset tab and typing the command `tab l`, `tab liability`, or
```
Comment (sheehui) [inline docs/UserGuide.md:132]: Assets Tab
You also forgot to add Liabilities tab after the tab commands

File: docs/UserGuide.md [hunk 6]
```diff
@@ -86,15 +86,87 @@ This marks the end of the quick start. Click
...
+`tab liabilites`.
+
+You will know that you are in the Asset tab if you see the **Assets** header below the Result Display or the Liability
```
Comment (sheehui) [inline docs/UserGuide.md:135]: Assets tab and Liabilities tab

File: docs/UserGuide.md [hunk 7]
```diff
@@ -86,15 +86,87 @@ This marks the end of the quick start. Click
...
+tab if you see the **Liabilities** header below the Result Display. You can see the total value of these assets/
+liabilities by looking at the number beside the **Total Asset/Liability Value**. 
+
+The Assets/Liabilities are displayed in a Table View where each row represents a new asset/liability, while each column
+represents a Field of an Asset/Liability.
```
Comment (sheehui) [inline docs/UserGuide.md:140]: not sure if field has to be in caps

File: docs/UserGuide.md [hunk 8]
```diff
@@ -86,15 +86,87 @@ This marks the end of the quick start. Click
...
+4. Asset/Liability Value
+5. Remarks
+
+Each asset/liability has an index which is sorted by ascending order, and if you would like to remove an existing
+asset/liability , you can keep track of the index by looking at the number to the left of the asset/liability name you
+would like to delete.
+
+The Asset/Liability tab also supports the `append` and `remove` command that allows you to add a new asset/liability or
+remove an existing asset/liability as shown 
+[here.](https://ay2122s1-cs2103t-w16-1.github.io/tp/UserGuide.html#4331-add-an-asset-to-a-client-append.)
```
Comment (sheehui) [inline docs/UserGuide.md:157]: tbh [here.](#4331-add-an-asset-to-a-client-append.) is sufficient

File: docs/UserGuide.md [hunk 9]
```diff
@@ -86,15 +86,87 @@ This marks the end of the quick start. Click
...
+
+#### 3.3.5 Notes Tab
+
+The Notes tab is the Client View tab that stores additional information about your client. This tab can be accessed by
```
Comment (sheehui) [inline docs/UserGuide.md:161]: Client Window tab? consider : The Notes tab stores additional information about your client.

File: docs/UserGuide.md [hunk 10]
```diff
@@ -86,15 +86,87 @@ This marks the end of the quick start. Click
...
+pressing on the `Notes button` when in any of the Client View tabs, or through typing the command `tab n`, `tab notes`
```
Comment (sheehui) [inline docs/UserGuide.md:162]: `Notes` button when you are in any of the Client Window tabs

**Review by sheehui — COMMENTED** [review]
good job donny and thanks for being speedy :)

File: docs/UserGuide.md (hunk 8 above)
Comment (DonnyS57) [reply docs/UserGuide.md:157]: wait the square brackets disappear in my preview ah

Comment (sheehui) [reply docs/UserGuide.md:254]: no like use #4331-add-an-asset-to-a-client-append inside the round brackets no need the http stuff

File: docs/UserGuide.md [hunk 11]
```diff
@@ -86,8 +86,39 @@ This marks the end of the quick start. Click
 
 ## 3. About
 
+This section aims to help familiarise you with how our application works as well as the terminologies that we use
+throughout our application and user guide.
+
 ### 3.1 Special Icons
 
+Here are the special icons used throughout the User Guide to help you utilise this guide more effectively.
```
Comment (sheehui) [inline docs/UserGuide.md:94]: our User Guide

**Review by sheehui — COMMENTED** [review]
just this one and the 2 unresolved comments above!

File: docs/UserGuide.md (hunk 2 above)
Comment (bharathcs) [reply docs/UserGuide.md:90]: also i believe its terminology

File: docs/UserGuide.md (hunk 8 above)
Comment (bharathcs) [reply docs/UserGuide.md:254]:
```suggestion
[here](#4331-add-an-asset-to-a-client-append).
```

File: docs/UserGuide.md (hunk 11 above)
Comment (bharathcs) [reply docs/UserGuide.md:94]:
```suggestion
Here are the special icons used throughout our User Guide to help you utilise this guide more effectively.
```

File: docs/UserGuide.md [hunk 12]
```diff
@@ -86,8 +86,39 @@ This marks the end of the quick start. Click
 
 ## 3. About
 
+This section aims to help familiarise you with how our application works as well as the terminologies that we use
```
Comment (bharathcs) [inline docs/UserGuide.md:89]:
```suggestion
This section aims to help familiarize you with how our application works as well as the terminology that we use
```

File: docs/UserGuide.md (hunk 2 above)
Comment (bharathcs) [reply docs/UserGuide.md:90]: I think its fine leh

File: docs/UserGuide.md [hunk 13]
```diff
@@ -86,8 +86,39 @@ This marks the end of the quick start. Click
...
+
+Tips are tidbits of helpful information that can help you have a better experience with DonnaFin.
+
+<div markdown="span" class="alert alert-primary">:bulb:
+Tip: Tips are helpful!
+</div><br>
+
+#### Notes
+
+Notes are here to provide you with extra information that they may find useful when using DonnaFin.
```
Comment (bharathcs) [inline docs/UserGuide.md:106]:
```suggestion
Notes are here to provide you with extra information that you may find useful when using DonnaFin.
```

File: docs/UserGuide.md [hunk 14]
```diff
@@ -86,8 +86,39 @@ This marks the end of the quick start. Click
...
+
+
+#### Caution 
+
+Cautions are around to warn you of potential unwanted side effects that may occur when using certain features of
+DonnaFin if you fail to follow this guide properly or tamper with the files of the application without adequate
+knowledge.
+
+<div markdown="span" class="alert alert-primary">
+:exclamation: Be wary of cautions!
```
Comment (bharathcs) [inline docs/UserGuide.md:119]:
```suggestion
:exclamation: Here lies a cautionary tale!
```

hehe a joke (yes shee hui, its lame)

**Review by bharathcs — APPROVED** [review]
LGTM!

**Pull request merged**